src/renderer/renderScene.ts

```diff
@@ -848,15 +860,23 @@ const renderTransformHandles = (
       if (renderConfig.selectionColor) {
         context.strokeStyle = renderConfig.selectionColor;
       }
+
       if (key === "rotation") {
+        context.fillStyle = oc.white;
         fillCircle(context, x + width / 2, y + height / 2, width / 2);
         // prefer round corners if roundRect API is available
       } else if (context.roundRect) {
+        if (cropModeEnabled) {
+          context.fillStyle = "#228be6";
```

Comment:
Did you mean to hardcode this?  (Line 878 also.)

Reply:
Yes and no.  I didn't know where else to put it.  The color fits both light and dark themes, so I considered it workable for now (whether that's right or wrong, I don't know).